src/JuliusSweetland.OptiKey.Core/Resources/Themes/Android_Dark.xaml

```diff
@@ -17,6 +17,7 @@ Copyright (c) 2019 OPTIKEY LTD (UK company number 11854839) - All Rights Reserve
         <ResourceDictionary Source="/OptiKey;component/Resources/Icons/KeySymbols.xaml" />
         <ResourceDictionary>
             <SolidColorBrush x:Key="WindowBorderBrush" Color="{StaticResource Grey}" />
+            <SolidColorBrush x:Key="KeyboardBackgroundBrush" Color="{StaticResource Black}" />
```

Comment:
Assuming this is only used by Dynamic Keyboards could this be renamed to DynamicKeyboardBackgroundBrush

Reply:
Also themes don't currently have a keyboard background, they instead set key, scratchpad, and question backgrounds. I presume this new brush overrides these somehow?

Reply:
I'm not 100% confident on exact behavior, but to make transparency possible, I had to set the MainWindow.xaml background color to Transparent. Otherwise, it would default to white behind a key that was supposed to be Transparent. So, because I made that change, I wanted to be sure the themes had a background color for empty areas where there might not be a key.

Reply:
The grid sections on dynamic keyboards used to use 'KeyDefaultBackgroundBrush', but I changed the code so the whole dynamic keyboard would use 'KeyboardBackgroundBrush'. The new name was for accuracy and consistency.

Reply:
Understood and agreed, although if the theme has a black keyboard background then 3 things should happen:
1. Standard keyboards should use this brush as their background
2. Dynamic keyboards should use this brush as their background
3. The background colour should be transparent to allow transparent keys? Or the brush needs to be settable from the dynamic keyboard.

Thoughts?

Reply:
1. Agree.
2. They do with the changes in this PR.
3. The brush is settable from the dynamic keyboard. I was overriding both the keyboard and key background color to achieve transparency.

Reply:
Here's some sample:

    <Keyboard>
    <Name>Demo</Name>
    <HideFromKeyboardMenu>False</HideFromKeyboardMenu>
	<BackgroundColor>Transparent</BackgroundColor>
	<BorderColor>Transparent</BorderColor>
    <Grid>
        <Rows>4</Rows>
        <Cols>5</Cols>
    </Grid>
    <Keys>
		<ActionKey><Row>0</Row><Col>0</Col><Height>1</Height><Width>1</Width><Action></Action><Label>Name: Demo</Label></ActionKey>
        <TextKey><Row>0</Row><Col>3</Col><Height>1</Height><Width>1</Width><Text>1</Text><ShiftUpLabel>1</ShiftUpLabel><ShiftDownLabel>!</ShiftDownLabel></TextKey>
        <DynamicKey><Row>0</Row><Col>4</Col><Height>1</Height><Width>1</Width><Label>Menu</Label><Symbol>MenuIcon</Symbol>
			<Steps><DestinationKeyboard>Menu</DestinationKeyboard><ReturnToThisKeyboard>false</ReturnToThisKeyboard></Steps>
		</DynamicKey>
        <ActionKey><Row>2</Row><Col>4</Col><Height>1</Height><Width>1</Width><Action>BackFromKeyboard</Action><Label>Back</Label><Symbol>BackIcon</Symbol></ActionKey>
		<ActionKey><Row>3</Row><Col>0</Col><Height>1</Height><Width>1</Width><Action>ClearScratchpad</Action><Label>Clear</Label><Symbol>ClearIcon</Symbol></ActionKey>
		<OutputKey><Row>3</Row><Col>1</Col><Height>1</Height><Width>3</Width><BackgroundColor>gray</BackgroundColor><Output>Scratchpad</Output><Label></Label></OutputKey>
		<ActionKey><Row>3</Row><Col>4</Col><Height>1</Height><Width>1</Width><Action>CopyAllFromScratchpad</Action><Label>Copy</Label><Symbol>CopyAllIcon</Symbol></ActionKey>
    </Keys>
</Keyboard>